packaging/standalone/src/tests/Neo4j-Management/unit/Get-Java.Tests.ps1

```diff
@@ -261,7 +261,25 @@ InModuleScope Neo4j-Management {
       It "should have correct Starting Class" {
         $resultArgs | Should Match ([regex]::Escape(' someclass'))
       }
-    }    
+    }
 
+	Context "Server Invoke - Should handle paths with spaces" {
+        $serverObject = global:New-MockNeo4jInstall -ServerVersion '3.0' -ServerType 'Community' `
+	      -RootDir 'TestDrive:\Neo4j Home' `
```

Comment:
Minor nit: Should these files be using 2 or 4 space indents?  Same for the other changes in this commit.